CHANGELOG

```diff
@@ -87,9 +87,13 @@
   - Add support for Ruby 2.3, 2.4, and 2.5 in compiled Windows binaries
   - Fix compatibility with libxcrypt [GH #164 by @besser82]
 
-[DRAFT] 4.0.0 MMM DD YYYY
+3.1.13 May 31 2019
```

Comment:
My change `- Start calibration from the minimum cost supported by the algorithm [GH #206 by @sergey-alekseev]` couldn't be included into this release. The release was on May 31st 2019, I submitted #206 on July 6th 2019.